src/Sylius/Bundle/AdminBundle/Resources/views/ProductVariant/Tab/_details.html.twig

```diff
@@ -13,6 +13,11 @@
             </div>
             {{ form_row(form.channelPricings) }}
         </div>
+        <div class="ui segment">
+            <div class="one field">
+                {{ form_row(form.shippingRequired) }}
```

Comment:
I guess it should be also added to the form template of simple product (it may result in a bug similar to #7735).